keyboards/handwired/wwa/rosetta/readme.md

```diff
@@ -0,0 +1,27 @@
+# rosetta
+
+![rosetta](https://i.imgur.com/KQL8cUe.jpg)
+
+21 Keys, 4 Fader MIDI Macro Keyboard
+
+* Keyboard Maintainer: [spacehangover](https://github.com/spacehangover)
+* Hardware Supported: Arduino Pro Micro, B103K Slide Pot
+* Hardware Availability: Arduino Pro Micro
```

Comment:
Is there a URL for either buying this keyboard or files to construct this keyboard?

Reply:
Not at the moment